src/browser/bounds_changed_state_tracker.js

```diff
@@ -139,58 +149,67 @@ function BoundsChangedStateTracker(uuid, name, browserWindow) {
     };
 
     const handleGroupedResize = (resizeChangeType, delta, originalWindowCachedBounds, windowToUpdate) => {
-        let { x, y, width, height } = windowToUpdate.browserWindow.getBounds();
-        if (resizeChangeType.height) {
-            if (delta.y) {
-                if (sharedBound(y, originalWindowCachedBounds.y)) {
-                    // resize windows with matching top bound
-                    y = toSafeInt(y + delta.y, y);
-                    height = height + delta.height;
-                }
-                if (sharedBound(y + height, originalWindowCachedBounds.y)) {
-                    // resize windows with matching bottom bound
-                    height = height - delta.height;
-                }
-            }
-            if (delta.y2) {
-                if (sharedBound(y + height, originalWindowCachedBounds.y + originalWindowCachedBounds.height)) {
-                    // resize windows with matching bottom bound
-                    height = height + delta.height;
-                }
-                if (sharedBound(y, originalWindowCachedBounds.y + originalWindowCachedBounds.height)) {
-                    // resize windows with matching top bound
-                    y = toSafeInt(y + delta.height, y);
-                    height = height - delta.height;
-                }
-            }
-        }
-        if (resizeChangeType.width) {
-            if (delta.x) {
-                if (sharedBound(x, originalWindowCachedBounds.x)) {
-                    // resize windows with matching left bound
-                    x = toSafeInt(x + delta.x, x);
-                    width = width + delta.width;
+        if (!trackingResize) {
+            return windowToUpdate.browserWindow.getBounds();
+        } else {
```

Comment:
Instead of making this a big `if...else` block, which makes the changeset look bigger than it is due to the indentation, just keep the `if` block.